File: even_or_odd.py
# def check_even_or_odd(number):
#     if number % 2 == 0:
#         print(f"{number} is even.")
#     else:
#         print(f"{number} is odd.")

# # Example usage:
# number = 4
# check_even_or_odd(number)

def check_even_or_odd(number):
    if number & 1 == 0:
        print(f"{number} is even.")
    else:
        print(f"{number} is odd.")

# Example usage:
number = 4
check_even_or_odd(number)

File: even_or_odd.js
// function checkEvenOrOdd(number) {
//    if (number % 2 === 0) {
//       console.log(number + " is even.");
//    } else {
//       console.log(number + " is odd.");
//    }
// }

// // Example usage:
// let number = 4;
// checkEvenOrOdd(number);

// function checkEvenOrOdd(number) {
//    if ((number & 1) === 0) {
//       console.log(number + " is even.");
//    } else {
//       console.log(number + " is odd.");
//    }
// }

// // Example usage:
// let number = 4;
// checkEvenOrOdd(number);

function checkEvenOrOdd(number) {
   if (Math.floor(number / 2) * 2 === number) {
      console.log(number + " is even.");
   } else {
      console.log(number + " is odd.");
   }
}

// Example usage:
let number = 4;
checkEvenOrOdd(number);
